po/pl.po

```diff
@@ -833,19 +866,19 @@ msgstr "Wystąpił nieznany błąd."
 
 #: ../src/main.cc:138
 msgid "Print information on the indicated element(s) to standard output"
-msgstr ""
+msgstr "Wydrukuj informacje wskazanym/ch pierwiastku/ach na wyjściu standardowym"
 
 #: ../src/main.cc:140 ../src/main.cc:148
 msgid "SYMBOL_OR_NUMBER"
-msgstr ""
+msgstr "SYMBOL_LUB_NUMER"
 
 #: ../src/main.cc:146
 msgid "Show the properties dialog for the indicated element(s)"
-msgstr ""
+msgstr "Pokaż okno z właściwościami na dla wskazanego/ych pierwiastka/ów"
```

Comment:
And an extra word here. Should be "Pokaż okno z właściwościami dla wskazanego/ych pierwiastka/ów"